### DISCUSSION TITLE
Download failed tools/tools-host

### DISCUSSION THREAD

**Discussion by Bars75** [Freetz-NG/freetz-ng#705] [Allgemein]
Hi!

_---> tools/tools-host ...
downloading ...
--2023-01-18 14:42:46--  https://freetz-ng.hackquest.com/tools-2023-01-15.tar.xz
Resolving freetz-ng.hackquest.com (freetz-ng.hackquest.com)... 82.165.251.131, 2001:8d8:1800:5fe::babe
Connecting to freetz-ng.hackquest.com (freetz-ng.hackquest.com)|82.165.251.131|:443... failed: Connection timed out.
Connecting to freetz-ng.hackquest.com (freetz-ng.hackquest.com)|2001:8d8:1800:5fe::babe|:443... failed: Network is unreachable.
Download failed - "https://freetz-ng.hackquest.com/tools-2023-01-15.tar.xz"  ->  error code 4

--2023-01-18 14:43:06--  https://freetz-ng.dropfiles.net/tools-2023-01-15.tar.xz
Resolving freetz-ng.dropfiles.net (freetz-ng.dropfiles.net)... failed: Name or service not known.
wget: unable to resolve host address ‘freetz-ng.dropfiles.net’
Download failed - "https://freetz-ng.dropfiles.net/tools-2023-01-15.tar.xz"  ->  error code 4

--2023-01-18 14:43:06--  https://freetz.magenbrot.net/tools-2023-01-15.tar.xz
Resolving freetz.magenbrot.net (freetz.magenbrot.net)... 31.172.113.113
Connecting to freetz.magenbrot.net (freetz.magenbrot.net)|31.172.113.113|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-01-18 14:43:06 ERROR 404: Not Found.

Download failed - "https://freetz.magenbrot.net/tools-2023-01-15.tar.xz"  ->  error code 8

--2023-01-18 14:43:06--  https://freetz.wirsind.info/tools-2023-01-15.tar.xz
Resolving freetz.wirsind.info (freetz.wirsind.info)... 188.165.115.52
Connecting to freetz.wirsind.info (freetz.wirsind.info)|188.165.115.52|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-01-18 14:43:07 ERROR 404: Not Found.

Download failed - "https://freetz.wirsind.info/tools-2023-01-15.tar.xz"  ->  error code 8

--2023-01-18 14:43:07--  https://raw.githubusercontent.com/Freetz-NG/dl-mirror/master/tools-2023-01-15.tar.xz
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.110.133, 185.199.111.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-01-18 14:43:07 ERROR 404: Not Found.

Download failed - "https://raw.githubusercontent.com/Freetz-NG/dl-mirror/master/tools-2023-01-15.tar.xz"  ->  error code 8
make[1]: *** [make/host-tools/tools-host/tools-host.mk:22: dl/tools-2023-01-15.tar.xz] Error 1
make: *** [Makefile:46: envira] Terminated_

tools-2023-01-15.tar.xz not avalible for download.

Thanks.

**Comment by fda77** [comment]
Dropfiles seems gone, hackquest seems unreliable, github has a quote of 1gb traffic permonth, the other mirror are not updated / have only pre freetz-ng stuff
So for this precompiled "tools" file you could disable `FREETZ_HOSTTOOLS_DOWNLOAD` in menuconfig and build for yourself. But currently it seems to work again

**Comment by Bars75** [comment]
Yes, thanks. I did so, before that there were no such difficulties with downloading.
